frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.test.ts

```diff
@@ -0,0 +1,89 @@
+import { PackagesRouter } from './PackagesRouter';
+
+const mockOrg: string = 'org';
+const mockApp: string = 'app';
+
+describe('PackagesRouter', () => {
+  describe('constructor', () => {
+    it('should initialize app and org properties', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+
+      expect(packagesRouter['app']).toEqual(mockApp);
+      expect(packagesRouter['org']).toEqual(mockOrg);
+    });
+
+    it('should default to empty strings if app and org are not provided', () => {
+      const routerWithoutParams = new PackagesRouter({});
+      expect(routerWithoutParams['app']).toEqual('');
+      expect(routerWithoutParams['org']).toEqual('');
+    });
+  });
+
+  describe('navigateToPackage', () => {
+    it('should navigate to the correct URL without subUrl', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+      const expectedUrl = `/editor/${mockOrg}/${mockApp}/overview`;
+
+      // Mock the window.location.assign method
+      const assignMock = jest.fn();
+      Object.defineProperty(window, 'location', {
+        value: { assign: assignMock },
+        writable: true,
+      });
+
+      packagesRouter.navigateToPackage('editorOverview');
+
+      expect(assignMock).toHaveBeenCalledWith(expectedUrl);
+    });
+
+    it('should navigate to the correct URL with subUrl', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+
+      const mockSubUrl = '?layout=123';
+      const expectedUrl = `/editor/${mockOrg}/${mockApp}/ui-editor${mockSubUrl}`;
```

Comment:
The portion of a URL following the "?" is referred to as query parameters. To enhance clarity, I suggest to rename the variable and update the description accordingly.

```suggestion

    it('should navigate to the correct URL and include queryParams', () => {
      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });

      const mockedQueryParams = '?layout=123';
      const expectedUrl = `/editor/${mockOrg}/${mockApp}/ui-editor${mockSubUrl}`;
```